### PR TITLE
minor changes to plotter and tensorboard services

### PR DESCRIPTION
**Pull request by fv316**

### CODE REVIEW

File: loaders/ecg_loader_bin.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+
+# class names and short hand class names
+classnames = ['Normal', 'Abnormal']
+sh_classnames = ['Normal', 'Abnormal']
+class_importance = np.array([1, 2])
+data_sets = ['mitbih_train.csv', 'mitbih_test.csv']
+
+
+class ECGLoader_bin(torch.utils.data.Dataset):
```
Comment (SzymonPajzert) [inline loaders/ecg_loader_bin.py]: Can't we somehow integrate that with the ECGLoader? It's copying code a bit and makes the masking to be duplicated as well
